File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int[] solution(int[] num_list) 
    {
        Array.Sort(num_list);
        List<int> removeList = new List<int>(num_list);
        for(int i=0; i<5; i++)
        {
            removeList.RemoveAt(0);
        }
        return removeList.ToArray();
    }
}

File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer(num_list.size()-5); // copy메소드를 쓸려면 벡터크기를 정의해야함
    sort(num_list.begin(), num_list.end());
    
    copy(num_list.begin()+5, num_list.end(), answer.begin());
    return answer;
}